src/opentelemetry.ml

```diff
@@ -669,6 +702,15 @@ end
 
     These metrics are emitted after each GC collection. *)
 module GC_metrics  = struct
+  (** See https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/process.md#process-runtimes *)
+  let runtime_attributes : Proto.Common.key_value list =
+    Conventions.Attributes.[
+        (Process.Runtime.name, "ocaml");
+        (Process.Runtime.version, Sys.ocaml_version);
+    ]
```

Comment:
Question: should we include these in all metrics/traces via `global_attributes`?

Reply:
sounds useful, but can we do it at the _batch_ level? seems super wasteful to emit these for each span/metrics